cmd/list.go

```diff
@@ -61,7 +62,9 @@ func formatListEntries(ls []apitypes.Entry) string {
 			name += "/"
 		}
 
-		table[i] = []string{name, mtimeStr, verbs}
+		mode := entry.Attributes.Mode().String()
```

Comment:
Hmm, why print mode? It isn't a very common attribute, at least not nearly as common as mtime.

If we do decide to print the mode, then I think this should mark nonexistent modes as `unknown`. That way, we don't imply to users that mode-less things like containers/VMs have a mode.

Reply:
It was useful when I wasn't getting reliable data out. I'll pull it back out.